walkthroughs/eks-canary-deployments-pipeline/shared_stack/lambda_functions/rollback_or_finish_upgrade/bootstrap

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
This file appears to be the exact same content as the `bootstrap` file above. Can we cut down on the copy/pasted content?

Reply:
It is possible. But each folder is related to a different AWS Lambda function and has all the resources needed for that function. Copying the file from the folder of the other function during bootstrap would work but could cause confusion if someone want's to customize the functions.